google/cloud/datastore_admin_v1/gapic/datastore_admin_client.py

```diff
@@ -43,10 +42,10 @@
 from google.longrunning import operations_pb2
 from google.protobuf import empty_pb2
 
-
-_GAPIC_LIBRARY_VERSION = pkg_resources.get_distribution(
-    "google-cloud-datastore",
-).version
+version = {}
```

Comment:
Is this being done to avoid importing `google.cloud.datastore` in the admin code?

Reply:
I am going to add a comment quick. This is odd enough that it is worth being clear ;)